spec/features/visitors/sign_up_spec.rb

```diff
@@ -82,24 +89,8 @@
       expect(current_path).to eq users_otp_path
     end
 
-    it 'disables OTP lockout during account creation' do
```

Comment:
I agree that we don't want an attempt counter. My point was that we want to test that we **don't** have an attempt counter. Since we have attempt counters elsewhere, we want to make sure they don't apply here.